### ISSUE TITLE
Allow using Hexagon with kernel.org LLVM

### ISSUE THREAD

**Issue by bhcopeland** [kernelci/tuxmake#216]
***In GitLab by @nathanchance on Mar 1, 2024, 22:04 GMT:***

Attempting to use Hexagon with `korg-clang` or `korg-llvm` results in:

```
$ tuxmake -a hexagon -k defconfig -r podman -t korg-clang-18 LLVM=1 default
E: Unsupported architecture/toolchain combination: hexagon/korg-clang-18
```

which should be allowed, as the kernel.org toolchains support Hexagon just like the Debian ones do.

**Assignees:** stylesen

*Migrated from GitLab: https://gitlab.com/Linaro/tuxmake/-/issues/216*

**Comment by bhcopeland** [comment]
***In GitLab by @stylesen on Mar 4, 2024, 12:44 GMT:***

Hi @nathanchance I am working on it. Will let you know once things are ready. You can see the initial MR here - https://gitlab.com/Linaro/tuxmake/-/merge_requests/372

**Comment by bhcopeland** [comment]
***In GitLab by @stylesen on Mar 6, 2024, 15:33 GMT:***

The support is available from tuxmake v1.23.2
